imports/api/utils.js

```diff
@@ -7,3 +7,7 @@ export function toggleElementInArray(collection, id, arrayName, element) {
   const modifier = { [action]: { [arrayName]: element } };
   collection.update(id, modifier);
 }
+
+export function deaccentLowerCase (text) { 
```

Comment:
O de jo, hogy van lett ilyen ujrahasznosithato fuggvenyunk!
Javaslom akkor a demo haz is hasznalhatja ezt. (Vagy lehet hogy mar FixtureBuilder-nek hivjak :)